notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2224 @@
+{
```

Comment:
<p>Change "Learn more here..." to:</p><p>Learn more about [<a href="https://cloud.google.com/vertex-ai/docs/general/locations#accelerators" target="_blank">hardware accelerator support for your region</a></p><p> </p><p>This style should be applied to all "learn more"s in the notebooks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>done</p>